maxim/logger/groq/client.py

```
@@ -0,0 +1,231 @@
+                        response = GroqHelpers.sync_stream_helper(response, generation, trace, is_local_trace)
+                    else:
+                        generation.result(GroqUtils.parse_completion(response))
+                        if is_local_trace and trace is not None:
+                            if response.choices and len(response.choices) > 0:
+                                trace.set_output(response.choices[0].message.content or "")
+                            else:
+                                trace.set_output("")
+                            trace.end()
+            except Exception as e:
+                scribe().warning(
+                    f"[MaximSDK][GroqInstrumentation] Error in logging generation: {e}"
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)